README.md

```diff
@@ -82,6 +84,10 @@ The passcode will expire after one week. Follow the steps below to sign in with
 
 UI tests require credentials that are only available to Twilio employees.
 
+## Languages
+
+This application was originally written in Java. However, all new code will be written in Kotlin and we are gradually converting existing Java code to Kotlin.
+
```

Comment:
I was actually suggesting the `Languages` section goes away completely since it is only 1 sentence. And suggesting it might be possible to nest the 1 sentence in another section. It doesn't really matter to me, just clarifying the original feedback.

Reply:
Oh I see. Yeah, I'm okay getting rid of it. I'll push that change up in a sec...